softether-linux-howto.md

```diff
@@ -79,6 +83,19 @@ Ping the remote gateway at 192.168.0.1 `ping 192.168.0.1 -c4`
 Check your public IP address `wget -qO- http://ipecho.net/plain ; echo` <- note that in this line, O is "capital letter O".
 The IP address returned should be your local public IP address.
 
+### Route issues on cloud computers
+
+If you are using a cloud Linux instance that you connect to over the Internet, be aware that the `dhclient` command will likely install a deafult route to the gateway received via DHCP over the VPN, which may cut off your ssh (or other) Internet connectivity to your cloud Linux instance.
```

Comment:
Suggest changing as follows:
Be aware that the `dhclient` command will install a default route...
If you are using a cloud Linux instance, you will loose your ssh (or other) Internet connectivity...